docusaurus/docs/e2e-test-a-plugin/use-the-api.md

```diff
@@ -0,0 +1,60 @@
+---
+id: api
+title: The API
+description: How to use the grafana/plugin-e2e API
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - e2e
+  - end-to-end
+  - API
+sidebar_position: 10
+---
+
+`@grafana/plugin-e2e` uses the [page object model](https://playwright.dev/docs/pom) pattern to simplify the authoring of tests and ease maintenance of the codebase. In the page object model, each web page of an application is represented as a class file.
+
+## Classes
+
+In the `@grafana/plugin-e2e` package, classes represent pages or components in Grafana. Their purpose is to encapsulate common UI operations in one place. They also handle UI deviations between different versions of Grafana.
+
+Classes are exported by the package, but they're are also exposed through the Playwright API via so called [fixtures](https://playwright.dev/docs/test-fixtures).
+
+## Fixtures
+
+`@grafana/plugin-e2e` defines a set of [custom fixtures](https://github.com/grafana/plugin-tools/blob/main/packages/plugin-e2e/src/types.ts/#L21-L258) that facilitates the end-to-end testing of Grafana plugins. In the following section, the different types of page fixtures are being explained.
+
+### Pages
+
+Page model objects can represent a new instance of a page or a page for an already existing resource. To see the full list of pages exposed by `@grafana/plugin-e2e`, refer to the Github [repository](https://github.com/grafana/plugin-tools/tree/main/packages/plugin-e2e/src/models/pages).
+
+#### Using a new, empty instance of a page type in a test
+
+To start a test in a new, empty page of a certain type, use the camel case representation of the page object model name.
+
+The following example uses the variable edit page. When using the `variableEditPage` fixture, the test will start with an empty variable edit form in a new dashboard.
+
+```ts
+test('test variable edit page', async ({ variableEditPage }) => {
+  await variableEditPage.setVariableType('Query');
+});
+```
+
+#### Using an existing resource
+
+To start a test with a page object model pointing to an aready existing resource, use any of the fixtures prefixed with `goto`.
```

Comment:
```suggestion
To start a test with a page object model that points to an aready existing resource, use any of the fixtures prefixed with `goto`.
```

Reply:
Grafana style prefer to avoid gerunds when possible because of translation difficulties.